### PR TITLE
New package: QuadSpinner.Gaea version 1.3.2.7

### PR DESCRIPTION
**Pull request by daxpandhi**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132402)

### CODE REVIEW

**Comment by daxpandhi** [comment]
@microsoft-github-policy-service agree [company="QuadSpinner"]

**Comment by daxpandhi** [comment]
@microsoft-github-policy-service agree company="QuadSpinner"

File: manifests/q/QuadSpinner/Gaea/1.3.2.7/QuadSpinner.Gaea.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: QuadSpinner.Gaea
+PackageVersion: 1.3.2.7
+InstallerType: nullsoft
+Installers:
+- Architecture: x64
+  InstallerUrl: https://get.gaea.app/Release/Gaea-1.3.2.7.exe
+  InstallerSha256: 301F1E0BAAC96B99A348F38FA9007961C468F6180BC518A9D9A65920C334AFD9
```
Comment (Trenly) [inline manifests/q/QuadSpinner/Gaea/1.3.2.7/QuadSpinner.Gaea.installer.yaml:10]:
```suggestion
  InstallerSha256: 301F1E0BAAC96B99A348F38FA9007961C468F6180BC518A9D9A65920C334AFD9
  ProductCode: Gaea
```
Product code is required as the publisher and version in ARP are both empty

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by daxpandhi — COMMENTED** [review]
I've made the suggested modification. Sorry for the oversight. :)

**Review by Trenly — APPROVED** [review]

**Pull request merged**